pkg/images/images.go

```diff
@@ -9,10 +9,20 @@ type Images []Image
 
 // Image defines the data we process about a docker image.
 type Image struct {
-	Name             string `yaml:"name"`
-	Comment          string `yaml:"comment,omitempty"`
-	OverrideRepoName string `yaml:"overrideRepoName,omitempty"`
-	Tags             []Tag  `yaml:"tags"`
+	Name             string       `yaml:"name"`
+	Comment          string       `yaml:"comment,omitempty"`
+	OverrideRepoName string       `yaml:"overrideRepoName,omitempty"`
+	Tags             []Tag        `yaml:"tags"`
+	Patterns         []TagPattern `yaml:"patterns"`
```

Comment:
Please keep this alphabetical only with `Name` as the first field.

The logic was that `Name` makes sense as a first field like in k8s and all following should be alphabetical. Unless you have some other reasoning for this order.

Reply:
Done